en/services/gimbal_v2.md

```diff
@@ -0,0 +1,282 @@
+
+
+### Message flow
+
+The ground station or any other user facing API such as an SDK are never to send messages directly to the gimbal device but always to the gimbal manager.
+However, the gimbal can and should broadcast its status to everyone, not just the gimbal manager.
+
+> **Note** The gimbal device must act only upon gimbal device messages that come from the gimbal manager!
+
+### Multiple gimbals
+
+Multiple gimbals per drone are supported.
+
+#### Component IDs
+
+Multiple component IDs are reserved for gimbal devices: `MAV_COMP_ID_GIMBAL`, `MAV_COMP_ID_GIMBAL2`, `MAV_COMP_ID_GIMBAL3`, `MAV_COMP_ID_GIMBAL4`, `MAV_COMP_ID_GIMBAL5`, `MAV_COMP_ID_GIMBAL6`.
+
+The listed component IDs should be used where possible (other ids may be used as long as the [MAV_TYPE](../messages/common.md#MAV_TYPE) is correctly set to [MAV_TYPE_GIMBAL](../messages/common.md#MAV_TYPE_GIMBAL)).
+
+#### Mapping from gimbal managers to gimbal devices
+
+Each gimbal manager needs to announce which gimbal it maps to.
+It is a 1:1 relationship, each gimbal manager is responsible for one gimbal device. However, multiple gimbal managers can be implemented on the same MAVLink component.
+E.g. an autopilot can implement two gimbal managers in order to control two gimbal devices.
+
+#### Addressing of gimbal devices
+
+Gimbal manager commands and messages have a param respective field to indicate the gimbal device component ID that they intend to control. However, the message needs to be sent to the gimbal manager. If all gimbal devices should be controlled this param/field can be set to 0 signalling "all".
+
+## Implementation and messages
+
+### Messages between ground station and gimbal manager
+
+#### Discovery of gimbal manager
+
+A ground station should initially discover all gimbal managers by sending [MAV_CMD_REQUEST_MESSAGE](#MAV_CMD_REQUEST_MESSAGE) to any sysid and compid. Every gimbal manager should respond with [GIMBAL_MANAGER_INFORMATION](#GIMBAL_MANAGER_INFORMATION).
```

Comment:
Note, I made all the links I can point down to my message summary table - which then has links to the message-in-detail. The reason is that this makes it easier to keep context if you need to.